components/pagination/style/index.less

```diff
@@ -27,9 +27,9 @@
     cursor: pointer;
     border-radius: @border-radius-base;
     user-select: none;
-    min-width: 28px;
-    height: 28px;
-    line-height: 28px;
+    min-width: @input-height-base;
```

Comment:
I think we need a new variable for this case, for Pagination is not a form control.